modules/@apostrophecms/modal/ui/apos/components/AposModalShareDraft.vue

```diff
@@ -175,15 +175,19 @@ export default {
       });
     },
     generateShareUrl(aposShareKey) {
-      const url = new URL(`${location.origin}${this.doc._url}`);
+      const url = new URL(this.doc._url);
```

Comment:
In my tests `this.doc._url` did not contain the origin, therefore `new URL()` was failing.
It was with a article

Reply:
`baseUrl` needs to be set at project level